path_generator:
  ros__parameters:
    init_pose_x: 0.0
    init_pose_y: 0.0
    init_pose_yaw: 0.0
    num_goals: 4
    x_goal_1: 1.47
    y_goal_1: 0.54
    x_goal_2: 1.29
    y_goal_2: 2.04
    x_goal_3: 0.05
    y_goal_3: 1.73
    x_goal_4: 0.0
    y_goal_4: 0.0
    # x_goal_1: 1.0
    # y_goal_1: -2.0
    # x_goal_2: 4.0
    # y_goal_2: -3.0
    # x_goal_3: 2.0
    # y_goal_3: 2.0
    # x_goal_4: 0.0
    # y_goal_4: 0.0

controller:
  ros__parameters:
    init_pose_x: 0.0
    init_pose_y: 0.0
    init_pose_yaw: 0.0

odometria:
  ros__parameters:
    init_pose_x: 0.0
    init_pose_y: 0.0
    init_pose_yaw: 0.0
    use_linear_model: false

aruco_detector_node:
  ros__parameters:
    init_pose_x: 0.0
    init_pose_y: 0.0
    init_pose_yaw: 0.0
